rama-socks5/src/server/udp/inspect.rs

```diff
@@ -0,0 +1,384 @@
+use std::fmt;
```

Comment:
you were on the socks5 feature branch it seems. Remove these files again please.